keps/sig-apps/2232-jobs-stopped/kep.yaml

```diff
@@ -0,0 +1,43 @@
+title: KEP Template
+kep-number: 2232
+authors:
+  - "@adtac"
+owning-sig: sig-apps
+status: provisional # provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+creation-date: 2020-01-06
+reviewers:
+  - "@soltysh"
+  - "@ahg"
+  - "@alculquicondor"
+  - TBD
+approvers:
+  - "@janetkuo"
+  - TBD
```

Comment:
add soltysh@ as an approver.

Reply:
soltysh is not  a KEP approver

Reply:
True, we need one of sig-apps leads to approve it.

Reply:
sg, I thought you are an approver.